region/maridia/outer.json

```diff
@@ -1095,6 +1128,135 @@
+                  "notable": true,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "mode": "any",
+                      "artificialMorph": false
+                    }},
+                    "canTrickyUseFrozenEnemies",
+                    "Wave",
+                    "h_canUseMorphBombs",
+                    {"or":[
+                      "Gravity",
+                      {"and":[
+                        "canSuitlessMaridia",
+                        "HiJump"
+                      ]}
+                    ]},
+                    {"or":[
+                      "SpringBall",
+                      "canBePatient",
+                      {"and":[
+                        "Gravity",
+                        "h_canCeilingBombJump",
+                        "h_canIBJ"
+                      ]}
+                    ]}
+                  ],
+                  "reusableRoomwideNotable": "Main Street G-Mode Overload Speed Blocks then use Frozen Crab",
+                  "note": [
+                    "Place bombs against the speed blocks until they are overloaded, then freeze a crab on the edge them such that Samus can jump inside and stand on the crab.",
+                    "Exit G-Mode and rotate to obtain the item, then kill the crab with Wave Beam to fall back down."
+                  ]
+                },
+                {
+                  "name": "G-Mode Morph IBJ PB Overload Speed Blocks Frozen Crab (From the Middle)",
+                  "notable": true,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "mode": "direct",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
+                    "Gravity",
+                    "canTrickyUseFrozenEnemies",
+                    "Wave",
+                    {"or":[
+                      "h_canArtificialMorphJumpIntoIBJ",
+                      "canDoubleBombJump",
+                      "canStaggeredIBJ",
+                      "canBePatient",
+                      {"enemyKill": {
+                        "enemies": [ [ "Skultera" ] ],
+                        "explicitWeapons": [ "PowerBomb" ]
+                      }}
+                    ]},
+                    {"ammo": { "type": "PowerBomb", "count": 5 }}
+                  ],
+                  "reusableRoomwideNotable": "Main Street G-Mode Overload Speed Blocks then use Frozen Crab",
+                  "note": [
+                    "This is a long climb, and getting around the fish under the missiles can be tricky or slow.",
+                    "Place 5 PBs on the corner of the ledge just up and left of the item to overload PLMs, then freeze a crab on the edge them such that Samus can jump inside and stand on the crab.",
```

Comment:
I was using the slope directly above the missile.  Could be anywhere on that ledge works.